app/services/calories/fetch_product_calories.rb

```diff
@@ -0,0 +1,29 @@
+module Calories
```

Comment:
I would put it in the same module as ApiClient, in Nutritionix. Those seems related